documentation/functions.md

```diff
@@ -764,6 +766,50 @@ Customers.WHERE(PERCENTILE(by=acctbal.ASC(), n_buckets=1000) == 1000)
 Regions.nations.customers.WHERE(PERCENTILE(by=acctbal.ASC(), levels=2) > 95)
 ```
 
+<!-- TOC --><a name="prev"></a>
+
+### PREV
+
+The `PREV` function returns the value of an expression from a preceding record in the collection. The arguments:
+
+- `expression`: the expression to return the shifted value of.
+- `n` (optional): how many records backwards to look (default: `1`).
+- `default` (optional): the value to output when there is no record `n` before the current record (default: `None`). This must be a valid literal.
+- `by`: 1+ collation values, either as a single expression or an iterable of expressions, used to order the records of the current context.
+- `levels` (optional): same `levels` argument as all other window functions.
+
+```py
+# Find the 10 customers with at least 5 orders with the largest average time
+# gap between their orders, in days.
+Customers.WHERE(COUNT(orders) > 5).CALCULATE(
+   name,
+   average_order_gap=DATEDIFF("days", PREV(order_date, by=order_date.ASC(), levels=1), order_date)
+).TOP_K(10, by=average_order_gap.DESC())
+
+# For every year/month, calculate the percent change in the number of
+# orders made in that month.
+PARTITION(
+   Orders(year=YEAR(order_date), month=MONTH(order_date)),
+   name="orders",
+   by=(year, month)
+).CALCULATE(
+   year,
+   month,
+   n_orders=COUNT(orders),
+   pct_change=
+      100.0
+      * (COUNT(orders) - PREV(COUNT(orders), by=(year.ASC(), month.ASC())))
+      / PREV(COUNT(orders), by=(year.ASC(), month.ASC()))
+)
+```
+
+<!-- TOC --><a name="next"></a>
+
+### NEXT
+
+The `NEXT` function is identical to `PREV` except that the direction is flipped, so `PREV(expr, n)` is the same as `NEXT(expr, -n)`.
```

Comment:
I suggest still writing down what is means and a simple example.
Description similar to this one line sentence from `PREV`
> The `PREV` function returns the value of an expression from a preceding record in the collection